go/service/config.go

```diff
@@ -117,6 +117,35 @@ func (h ConfigHandler) GetExtendedStatus(ctx context.Context, sessionID int) (re
 	return libkb.GetExtendedStatus(libkb.NewMetaContext(ctx, h.G()))
 }
 
+func (h ConfigHandler) GetAllProvisionedUsernames(ctx context.Context, sessionID int) (res keybase1.AllProvisionedUsernames, err error) {
+	defaultUsername, all, err := libkb.GetAllProvisionedUsernames(libkb.NewMetaContext(ctx, h.G()))
+	if err != nil {
+		return res, err
+	}
+
+	// If the default is missing, fill it in from the first provisioned.
+	if defaultUsername.IsNil() && len(all) > 0 {
+		defaultUsername = all[0]
+	}
+
+	// Callers expect ProvisionedUsernames to contain the DefaultUsername, so
+	// we ensure it is here as a final sanity check before returning.
```

Comment:
i'm surpised `GetAllProvisionedUsernames` doesn't include the current one.  Why not?

Reply:
https://github.com/keybase/client/blob/master/go/libkb/config.go#L312

Reply:
now i remember. thanks!